mbq/metrics/__init__.py

```diff
@@ -73,10 +73,9 @@ def namespace(self):
             )
         return namespace
 
-    @property
-    def tags(self):
+    def make_tags(self):
```

Comment:
Changed this to a method because it went against my expectations of what a property should be doing. It's important that this creates a copy, since the functions calling it then mutate its return value. I generally expect properties to a) do very little work if any, and b) return the same object if I call them multiple times. Making this a method and including the word `make` indicates that it's doing meaningful work.